lychee-bin/src/commands/check.rs

```diff
@@ -46,7 +46,7 @@ where
     let cache = params.cache;
     let accept = params.cfg.accept;
 
-    let pb = if params.cfg.no_progress {
+    let pb = if params.cfg.no_progress || params.cfg.verbose.log_level() >= log::Level::Info {
```

Comment:
This is here, because showing the progress bar in verbose mode makes the output harder to read. It's not related to the other changes in this PR, but I got annoyed by the output while troubleshooting, so I fixed it in one go.